libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -30,15 +48,101 @@ export class ChartJSService {
       dataLabels,
       customOptions,
       annotations,
+      datalabelOptions,
       highlightedElements,
     } = args;
 
-    const datasets = this.createDatasets(data, highlightedElements);
+    const suffix = datalabelOptions?.valueSuffix ? datalabelOptions.valueSuffix : '';
+
+    // We need to modify the datasets in order to add tooltips.
+    const datasets =
+      datalabelOptions?.showCurrent || datalabelOptions?.showMax || datalabelOptions?.showMin
+        ? this.createDatasets(this.addTooltips(data, datalabelOptions, suffix), highlightedElements)
+        : this.createDatasets(data, highlightedElements);
+
     const options = this.createOptionsObject({ type, customOptions, annotations });
-    const config = this.createConfigurationObject(type, datasets, options, dataLabels);
+    let config = this.createConfigurationObject(type, datasets, options, dataLabels);
+
+    if (type === ChartTypes.stock) {
+      config.options.plugins.tooltip.callbacks.label = (context) => {
+        return context.formattedValue + suffix;
+      };
+
+      config.options.scales.y.ticks.callback = (value) => {
+        return value + suffix;
+      };
+
+      // assuming that the first dataset controls the datespan.
+      const chartPeriod = this.chartConfigService.findChartPeriod(datasets[0]);
+      config.options = this.handleLocalization(
+        config.options,
+        chartPeriod,
+        datalabelOptions.locale
+      );
+    }
+
     this.initializeNewChart(targetElement.nativeElement, config);
   }
 
+  private getLocale(key: string) {
+    return { 'da-DK': da, 'en-US': enUS }[key];
+  }
+  private getDisplayFormats(key: string) {
+    return {
+      'da-DK': {
+        hour: 'HH:mm',
+        day: 'd MMM',
+      },
+      'en-US': {
+        hour: 'HH:mm',
+        day: 'MMM d',
+      },
+    }[key];
+  }
+
+  private handleLocalization(options, chartPeriod, language): ChartOptions {
+    language = language || 'en-US';
+    // Handle localization in graph.
+    options.locale = language;
+
+    // Update chart options with the given period.
+    const scaleX = options.scales.x as any;
```

Comment:
I tried to remove `as any` and it seemed to work just fine :D 
```suggestion
    const scaleX = options.scales.x;
```